docs/maintenance/uninstall.md

```diff
@@ -7,7 +7,6 @@ To do so run:
 helm -n codacy uninstall codacy
 kubectl -n codacy delete --all pod &
 kubectl -n codacy delete --all pvc &
-kubectl -n codacy delete --all pv  &
 kubectl -n codacy delete --all job &
 sleep 5
 kubectl -n codacy patch pvc -p '{"metadata":{"finalizers":null}}' $(kubectl -n codacy get pvc -o jsonpath='{.items[*].metadata.name}')
```

Comment:
Also those:

kubectl -n codacy patch pv -p '{"metadata":{"finalizers":null}}' $(kubectl -n codacy get pv -o jsonpath='{.items[*].metadata.name}')

kubectl -n codacy get pv  &

Maybe we can add a step for people to run `kubectl get pv | grep codacy` and delete codacy's PVs

Reply:
For now, I removed the missing lines.

If deleting the PVs in two steps is the way to go (fetching information about the PVs associated with Codacy first and then deleting them), feel free to add this solution on this pull request.